### ISSUE TITLE
Functionality for Stein variational policy gradient and/or regularization through the use of prior policies

### ISSUE THREAD

**Issue by leonhalgryn** [facebookresearch/Pearl#44]
Are there any plans to add functionality to allow using prior policies for regularization similar to that of the Stein variational policy gradient (SVPG) (SVPG paper available at: https://arxiv.org/abs/1704.02399)

**Comment by GreatArcStudios** [comment]
Yeah this would be pretty great if added. Looks like as of now they'd have to add it on a per algorithm basis as it is a modification to the loss function. Perhaps they could abstract away the loss into a module.

**Comment by rodrigodesalvobraz** [comment]
Thank you. This would be good but we are currently working on higher-priority items. We will leave the issue open and update it when we get to it. Thank you.
